src/cmake/dependencies.cmake

```diff
@@ -350,14 +350,33 @@ if(NOT USE_SYSTEM_MAGNUM)
     # for Python as well; and reset that back to strange build procedures that
     # turn some features off again later can still work.
     if(BUILD_GUI_VIEWERS)
-      set(MAGNUM_PYTHON_BINDINGS_STATIC_PLUGINS
-          MagnumPlugins::StbTrueTypeFont Magnum::AnySceneImporter
-          MagnumPlugins::AssimpImporter CACHE STRING "" FORCE
+      set(
+        MAGNUM_PYTHON_BINDINGS_STATIC_PLUGINS
+        MagnumPlugins::StbTrueTypeFont
+        Magnum::AnySceneImporter
+        Magnum::AnyImageImporter
+        Magnum::AnyImageConverter
+        MagnumPlugins::StbImageImporter
+        MagnumPlugins::StbImageConverter
+        MagnumPlugins::BasisImporter
+        MagnumPlugins::StanfordImporter
+        MagnumPlugins::GltfImporter
+        MagnumPlugins::AssimpImporter
+        CACHE STRING "" FORCE
       )
     else()
-      set(MAGNUM_PYTHON_BINDINGS_STATIC_PLUGINS Magnum::AnySceneImporter
-                                                MagnumPlugins::AssimpImporter
-          CACHE STRING "" FORCE
+      set(
+        MAGNUM_PYTHON_BINDINGS_STATIC_PLUGINS
+        Magnum::AnySceneImporter
+        MagnumPlugins::AssimpImporter
+        Magnum::AnyImageImporter
+        Magnum::AnyImageConverter
+        MagnumPlugins::StbImageImporter
+        MagnumPlugins::StbImageConverter
+        MagnumPlugins::BasisImporter
+        MagnumPlugins::StanfordImporter
+        MagnumPlugins::GltfImporter
+        CACHE STRING "" FORCE
```

Comment:
Hmm, now that I see it in full, it might make sense to put these in a common variable so future additions don't need to happen to two places:

```cmake
set(common_plugins
  Magnum::AnyImageConverter
  Magnum::AnyImageImporter
  Magnum::AnySceneImporter
  MagnumPlugins::AssimpImporter
  MagnumPlugins::BasisImporter
  MagnumPlugins::GltfImporter
  MagnumPlugins::StanfordImporter
  MagnumPlugins::StbImageConverter
  MagnumPlugins::StbImageImporter)
if(BUILD_GUI_VIEWERS)
  set(MAGNUM_PYTHON_BINDINGS_STATIC_PLUGINS 
    ${common_plugins}
    MagnumPlugins::StbTrueTypeFont
    CACHE STRING "" FORCE)
else()
  set(MAGNUM_PYTHON_BINDINGS_STATIC_PLUGINS 
    ${common_plugins}
    CACHE STRING "" FORCE)
endif()
```

Reply:
Yep, makes sense. Thanks for the code.